sdk/tables/Azure.Data.Tables/tests/SessionRecords/TableClientLiveTests/EntitiyCanBeUpserted.json

```diff
@@ -51,7 +51,7 @@
       "RequestHeaders": {
         "Authorization": "Sanitized",
         "Content-Length": "88",
-        "Content-Type": "application/json; odata=nometadata",
+        "Content-Type": "application/json",
```

Comment:
Why did these need to change?

Reply:
Currently headers are not compared in playback mode for matching record for track 2, this is a bug.  This PR is fixing this bug, these tests fail without these changes since 'Content-Type' is `application/json` in request during CI. Data sdk team should record these test cases after merging this PR.

Reply:
I confirmed with your branch that re-recording maintains the header as you have it here.